src/onegov/org/forms/form_registration.py

```diff
@@ -204,8 +204,9 @@ def validate_limit(self, field: IntegerField) -> None:
         if not self.limit_attendees.data:
             return
 
-        assert self.limit.data is not None
-        if self.limit.data < self.claimed_spots:
+        assert self.limit.data is not None  # but may be 0 / limit inactive
```

Comment:
Is this only for typing or does this (being 0) actually occur? Then we should probably also raise a ValidationErro

Reply:
Yes it can happened, https://linear.app/onegovcloud/issue/OGC-1557.

Reply:
Actually it cannot happen from the user side, it could happen if a programmer does a mistake as the value will be set to 0 or the limit provided in the form during `populate_obj`
https://github.com/OneGov/onegov-cloud/blob/master/src/onegov/org/forms/form_registration.py#L131

So I guess I will keep the assert statement not switching to a ValidationError